### ISSUE TITLE
IDOR : User Can Create MDF Fund Request for the plan not assigned to his Organization using api/SaveMDFRequest

### ISSUE THREAD

**Issue by IshantRevclerx** [MindMatrix/SecurityTracker#85]
URL : https://main.dvl.amp.vg/#manage/mdf/funds
User : Security Ishant
Company Name : Automation Testers

STR : 
  In the post request of api/SaveMDFRequest, replaced the activity id with activity id of any fund plan which is not assigned to his organization. and send the request.

Actual Result : 
MDF Fund Request will be send for the fund plan which his not asssigned to his organization.

Fund Plan : Testing Security is asigned to security-1 Organization.
![Testing Security-1](https://user-images.githubusercontent.com/92088937/180749819-2e213e66-8e0e-4797-a10b-2c5b0103951e.png)

Fund Plan Request is created by user who is in Automation Organization.
![Fund Plan Request--Security Ish](https://user-images.githubusercontent.com/92088937/180750111-059573e3-53e1-41b6-84e8-a1781903c56c.png)

Expected Result : 
User Should not be able to create Request to other organization fund plan.

**Comment by Vish-Patil** [comment]
https://github.com/MindMatrix/AMP/pull/51735

**Comment by Vish-Patil** [comment]
@sunrao please get this tested in following PR #51735

**Comment by IshantRevclerx** [comment]
@Vish-Patil Sir, I have checked above security issue on the given branch

**Comment by IshantRevclerx** [comment]
@Vish-Patil Sir, I have checked above security issue on the given branch ( https://pr-51735.dvl.amp.vg/ ) and its working as expected.
But, the grid is not horizonatly scrolabel and some of the data is moving out of the page.
similar issue is on the other pages of MDF as well.
![image](https://user-images.githubusercontent.com/92088937/186583591-69f02173-762a-45a0-9bff-b3111f0521ab.png)

User Name : Security Ishant
Company Name : Automation Testers.

**Comment by Vish-Patil** [comment]
@IshantRevclerx please check now.

**Comment by IshantRevclerx** [comment]
@Vish-Patil Sir, above Issue is still there on MDF Approval Request Page( Partial Reimbursement, Reject Request) and MDF Fund Request page ( Rejected Request).

![image](https://user-images.githubusercontent.com/92088937/186828652-1a95f558-2eac-4fb1-9424-089b477ad8d0.png)

In the above screenshoot "Updated on" column moved out of the page and there is no horizantal scrol available to see the column.

**Comment by Vish-Patil** [comment]
please check

**Comment by IshantRevclerx** [comment]
@Vish-Patil Sir, above Issue is still there on MDF Approval Request Page( Partial Reimbursement, Reject Request, Total Reimbursment Request) on given branch (https://pr-51735.dvl.amp.vg/ )
![image](https://user-images.githubusercontent.com/92088937/186927156-b36eaf22-8526-4e17-9f99-f76692ebec4f.png)

**Comment by Vish-Patil** [comment]
@IshantRevclerx is this issue only on this branch or on main as well?

**Comment by IshantRevclerx** [comment]
@Vish-Patil sir, The above UI Issue is on the main as well.

**Comment by shivmishra** [comment]
@sunrao @IshantRevclerx @shaileshtayde  if API issue fixed then for usability we can create new issue from main instead security issue

**Comment by Vish-Patil** [comment]
yes as this issue is due to screen size as no one reported it on live yet.

**Comment by IshantRevclerx** [comment]
@Vish-Patil sir, @sunrao sir, , I have checked above security issue on the given branch ( https://pr-51735.dvl.amp.vg/ )and user is not able to create MDF Fund Request for the plan not assigned to his Organization.

For the UI, an issue (https://github.com/MindMatrix/AMP/issues/51769) is already created by Priyank on the main.

**Comment by IshantRevclerx** [comment]
@Vish-Patil sir, I have checked the above issue on Main as the PR was merged and the issue is getting reproduced on Main.
